platform/broadcom/saibcm-modules/debian/opennsl-modules.init

```diff
@@ -32,14 +32,16 @@ function create_devices()
 #                                 level logs
 function load_kernel_modules()
 {
-    modprobe linux-kernel-bde dmasize=32M maxpayload=128 debug=4 dma_debug=1
+    modprobe linux-kernel-bde dmasize=128M maxpayload=128 debug=4 dma_debug=1 usemsi=1
```

Comment:
is usemsi working for all platforms?

Reply:
128M is better to be platform specific as well.

Reply:
I have reverted the new change of 128M and usemsi flag for now for the first release of SAI 3.7. Will do additional change using platform templates as a follow on commit so that it will support multi ASIC platforms as well.

Reply:
I have updated the pull request with the change as discussed so that the variables like dmasize, usemsi flag could be taken from a platform specific env file.

Reply:
please consider aboot_platform.

Reply:
Done